app/src/main/java/com/example/georgesamuel/dubaihotels/viewModel/HotelViewModel.java

```diff
@@ -0,0 +1,54 @@
+package com.example.georgesamuel.dubaihotels.viewModel;
+
+import android.app.Application;
+import android.util.Log;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.LiveData;
+import androidx.lifecycle.MutableLiveData;
+
+import com.example.georgesamuel.dubaihotels.model.Hotel;
+import com.example.georgesamuel.dubaihotels.model.HotelsResponse;
+import com.example.georgesamuel.dubaihotels.remote.ClientAPI;
+import com.example.georgesamuel.dubaihotels.remote.RetrofitClient;
+
+import java.util.List;
+
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+
+public class HotelViewModel extends AndroidViewModel {
+
+    private MutableLiveData<List<Hotel>> hotelsList = new MutableLiveData<>();
+    private ClientAPI clientAPI;
+    private final String TAG = "HotelViewModel";
```

Comment:
Use   private static final String TAG = MyClass.class.getSimpleName(); to define TAG